reload/dpl-platform-poc
# DDFDP-7 - Initial Terraform and doc setup

#### What does this PR do?
Introduces the basic Terraform setup we'll be using going forward. 
It has automation for setting up the base resources Terraform needs in order to function, and sets up our first POC environment.

Terraform is to be run from a docker-based shell, so that is included as well.

Last but not least I've included a documentation setup auto-renderes plantuml/drawio diagrams when they are comitted.

#### How should this PR be reviewed?
Look trough the commits, I've tried to keep them seperate and they should give a good overview of what seperate features has been added.

#### How should this be manually tested?
The terraform setup _should_ work, so - check out the branch, cd to `infrastructure/terraform/alpha/poc01` and launch the shell (put dplsh.sh on your path, or do a `../../../../tools/dplsh/dplsh.sh`

That should switch you to the correct terraform workspace.

#### What are the relevant tickets?
DDFDP-7
